clients/include/rocblas_datatype2string.hpp

```diff
@@ -129,42 +144,34 @@ constexpr rocblas_side char2rocblas_side(char value)
     }
 }
 
-constexpr rocblas_datatype char2rocblas_datatype(char value)
```

Comment:
We no longer need to convert `char` to `rocblas_datatype` anywhere -- it's all done with `string` now.